mypy/semanal.py

```diff
@@ -4362,23 +4362,6 @@ def create_getattr_var(self, getattr_defn: SymbolTableNode,
             return v
         return None
 
-    def lookup_fully_qualified(self, name: str) -> SymbolTableNode:
```

Comment:
In the above comment the idea would be to have the simple implementation that calls `lookup_fully_qualified_or_none` and does an assert here.